### PR TITLE
Wire the repo up for expeditor

### PR DESCRIPTION
**Pull request by tas50**
Automate how we ship this gem.

### CODE REVIEW

File: .expeditor/config.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Documentation available at https://expeditor.chef.io/docs/getting-started/
+---
+# Slack channel in Chef Software slack to send notifications about build failures, etc
+slack:
+  notify_channel: sustaining-notify
```
Comment (tduffield) [inline .expeditor/config.yml]: like this: 

```
slack:
  notify_channel:
    - sustaining-notify
    - chef-notify
```

**Comment by tas50** [comment]
@tduffield Fixed

**Comment by tas50** [comment]
@tduffield any idea about the config validation warning?

**Comment by tduffield** [comment]
@tas50 yeah, it's a regression that only appears the first time you add a changelog. I'll try and fix it but you should be good to merge

**Comment by tas50** [comment]
Thanks for the heads up

**Pull request merged**